fineract-provider/src/main/resources/sql/migrations/core_db/V362__reschedule_interest_to_lrs.sql

```diff
@@ -0,0 +1,22 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements. See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership. The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License. You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied. See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+--
+
+
+alter table m_loan_repayment_schedule
+    add column `reschedule_interest_portion` decimal(19,6) DEFAULT NULL after `accrual_interest_derived`;
```

Comment:
Where are the changes to schedule history table?

Reply:
The history table need not be altered, as the interest portion is carried from the current schedule to the new schedule.